src/js/app.js

```diff
@@ -73,29 +74,52 @@ jQuery(document).ready(function() {
             return true;
         }
 
+        history.pushState({page:$(this).attr("href")}, null, $(this).attr("href"));
+
+        previous_url = $(this).attr("href");
         $("#docsContent").load($(this).attr("href"));
         $(window).animate({ scrollTop: 0 }, "slow");
         //$("#docsContent").html('hello');
         // $("#docsContent").load('install');
 
-        //
         // Prevent browsers default behavior to follow the link when clicked
         $('#sidebar').find("a").removeClass('active');
         $(this).addClass('active');
-
+        
         return false;
     });
 
-  /*
-   * Handle links in "docsContent" so that relative links are opened in right pane and external are opened
-   *  in a new window.
-   *
-   *  NOTE: jQuery.load() is used to load the content in the #docsContent div.  In order to capture this event
-   *      $on('click', ...) function needs to be used so that added elements after initial document load
-   *      are handled.   See http://api.jquery.com/on/ "Delegated events" for more details.
-   *
-   *     Also see: https://stackoverflow.com/questions/14339309/jquery-click-event-handlers-dont-work-after-loading-html-page-with-load for details
-  */
+
+    $(window).on('popstate', function(e) {
+        if (e.originalEvent.state !== null) {
+//            console.log("popstate current: ", e.originalEvent.state.page);
+//            console.log("previous is : ", previous_url);
+            $('#sidebar').find("#" + previous_url).removeClass('active');
```

Comment:
The previous_url is the state prior to the current popped state. Googling didn't show an obvious way to get the previous state, so used the global to store previous states.